x-pack/platform/packages/shared/kbn-inference-endpoint-ui-common/src/components/configuration/authentication_form_items.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useMemo, useState } from 'react';
+import { EuiButtonGroup, EuiFlexGroup, EuiFlexItem, EuiTitle } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { ConfigEntryView } from '../../types/types';
```

Comment:
nit: import type ConfigEntryView

Reply:
Good catch - updated in 7425aa042767c23c6083a2d7ee528382e2dcfcef